doc/source/using.rst

```diff
@@ -25,6 +31,23 @@ template. No pre-processing is done on the data, so the template needs
 to handle aspects like ``None`` values and fields that have values
 that may interfere with parsing reStructuredText.
 
+If you set `:key: any_key` on the directive,
+you will have a top-level ``key`` variable.
+This can be used to render a subset of the data in the template.
+
+If you set `:include_context:` on the directive,
+you template will also have access to the builder's ``html_context`` while rendering.
+This will be in the global template context, not inside ``data``.
```

Comment:
We should probably also make it clear that any names in the html_context will not be able to override names used by the extension (like "data", "key", etc.) so that the order of precedence is clear.